sys/include/net/gnrc/lorawan/region.h

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright (C) 2019 HAW Hamburg
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup net_gnrc_lorawan
+ * @{
+ *
+ * @file
+ * @brief   GNRC LoRaWAN region specific functions
+ *
+ * @author  José Ignacio Alamos <jose.alamos@haw-hamburg.de>
+ */
+#ifndef NET_GNRC_LORAWAN_REGION_H
+#define NET_GNRC_LORAWAN_REGION_H
+
+#include "net/gnrc/lorawan.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#define GNRC_LORAWAN_LC_1 (868100000)   /**< First default EU868 channel */
```

Comment:
maybe use a array here, which might simplify usage and extension, i.e.:
```
static const uint32_t gnrc_lorawan_default_channels[] = {
    868100000UL,
    868300000UL,
    868500000UL
}
```

Reply:
done